### ISSUE TITLE
Ability to drag tabs out of existing terminal window (similar behaviour to what we experience in web browsers)

### ISSUE THREAD

**Issue by 3SCadmin** [microsoft/terminal#16031]
I'm probably not the first person to request this, but as we do not yet have this functionality it warrants repeating.

# Description of the new feature/enhancement

Ability to drag tabs out of existing terminal window 
The goal is to provide 2 new behaviours:

- Ability to drag a tab out of it's original window, thereby creating a new window.
- Ability to drag a tab out of it's original window, moving it to another (existing) window,

Basically mimicking the current behaviour web browsers offer.

**Comment by lhecker** [comment]
Thanks for the suggestion! This was actually already being tracked by another issue on our repo. See: #1256
It ships in Windows Terminal Preview 1.18 and will soon arrive in non-Preview as well. 🙂

/dup #1256

**Comment by 3SCadmin** [comment]
>> It ships in Windows Terminal Preview 1.18

Thank you for the tip.
Moved everything over to 1.18, and got it to work.
